edns.go

```diff
@@ -286,6 +289,32 @@ func (e *EDNS0_SUBNET) String() (s string) {
 	return
 }
 
+// The Cookie EDNS0 option
+//
+//o := new(dns.OPT)
```

Comment:
Ah yes, looks like you fixed the spaces already. Let me know if you wanted me to do anything else.